autoprof/image/image_object.py

```diff
@@ -107,40 +136,56 @@ def shape(self):
         """
         return self.window.shape
     @property
-    def center(self):
+    def center(self)-> torch.Tensor:
         """
         Returns the center of the image window.
 
         Returns:
             torch.Tensor: A 1D tensor of shape (2,) containing the (x, y) coordinates of the center.
         """
         return self.window.center
-    def center_alignment(self):
-        """Determine if the center of the image is aligned at a pixel center
-        (True) or if it is aligned at a pixel edge (False).
+    def center_alignment(self)-> torch.Tensor:
+        """Determine if the center of the image is aligned at a pixel center (True) 
+           or if it is aligned at a pixel edge (False).
 
         """
-        return torch.isclose(((self.center - self.origin) / self.pixelscale) % 1, torch.tensor(0.5, dtype = AP_config.ap_dtype, device = AP_config.ap_device), atol = 0.25)
+        return torch.isclose(((self.center - self.origin) / self.pixelscale) % 1, torch.Tensor(0.5, atol = 0.25, dtype = AP_config.ap_dtype, device = AP_config.ap_device))
```

Comment:
Looks like the `atol = 0.25` argument has been put inside the tensor creation. It should come after the tensor is created